static/app/views/releases/detail/overview/index.tsx

```diff
@@ -516,17 +709,28 @@ class ReleaseOverview extends AsyncView<Props> {
                       withChart
                     />
                     <Feature features={['performance-view']}>
-                      <TransactionsList
-                        location={location}
-                        organization={organization}
-                        eventView={releaseEventView}
-                        trendView={releaseTrendView}
-                        selected={selectedSort}
-                        options={sortOptions}
-                        handleDropdownChange={this.handleTransactionsListSortChange}
-                        titles={titles}
-                        generateLink={generateLink}
-                      />
+                      {hasReleaseComparison ? (
```

Comment:
Can you please guard this with a different feature flag?
`release-comparison` flag is already enabled for early adopters and I reckon we will want to play with this performance stuff for a while before releasing it to customers.